private/buf/bufcli/flags_args.go

```diff
@@ -17,6 +17,7 @@ package bufcli
 import (
 	"errors"
 	"fmt"
+	"slices"
```

Comment:
FYI - I had this waiting a while but noticed that formatting wasn't being enforced properly so wanted to fix https://github.com/bufbuild/buf/pull/3774 first.